blockchain/bench_test.go

```diff
@@ -101,8 +101,9 @@ func genValueTx(nbytes int) func(int, *BlockGen) {
 	return func(i int, gen *BlockGen) {
 		toaddr := common.Address{}
 		data := make([]byte, nbytes)
-		gas, _ := types.IntrinsicGas(data, false, false)
-		signer := types.NewEIP155Signer(params.TestChainConfig.ChainID)
+		chainConfig := &params.ChainConfig{ChainID: big.NewInt(0)}
```

Comment:
Why is this chain config updated? It seems that the chainId  and unitprice is differernt from `TestChainConfig`?

Reply:
Thanks for the comment! I reverted to use `TestChainConfig`. The reason why I changed to use own chain config is that `TestChainConfig` can be accessed globally. However, it is not a problem here, so I reverted.